webapp/sources/pom.xml

```diff
@@ -432,44 +432,44 @@ limitations under the License.
          parser option - see https://github.com/scala/scala-xml/releases/tag/v2.1.0 -->
     <scala-xml-version>1.3.0</scala-xml-version>
     <lift-version>4.0.0-M1</lift-version>
-    <slf4j-version>2.0.13</slf4j-version>
-    <logback-version>1.5.6</logback-version>
-    <jodatime-version>2.12.7</jodatime-version>
-    <jodaconvert-version>2.2.3</jodaconvert-version>
-    <commons-io-version>2.16.1</commons-io-version>
-    <commons-lang-version>3.15.0</commons-lang-version>
+    <slf4j-version>2.0.16</slf4j-version>
+    <logback-version>1.5.12</logback-version>
+    <jodatime-version>2.13.0</jodatime-version>
+    <jodaconvert-version>3.0.1</jodaconvert-version>
+    <commons-io-version>2.17.0</commons-io-version>
+    <commons-lang-version>3.17.0</commons-lang-version>
     <commons-text-version>1.12.0</commons-text-version>
     <commons-codec-version>1.17.0</commons-codec-version> <!-- can't update to 1.17.1 because of md5 requiring a non-empty salt prefix -->
```

Comment:
1.17.1 is still the latest one available, but at some point we will want to upgrade because of the [library change](https://github.com/apache/commons-codec/commit/ad28b0f1ea4010dc0eaeb1d1cda041efe0ef7323#diff-76d1349af6676238640c2ababc633639294e8740b10b10ece941221c28c79585R281), either : 
- by prepending the `$1$` prefix [when hashing with our custom smd5 hash format](https://github.com/Normation/rudder/blob/master/webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala#L113) , that means that our md5 hashes for AIX will have a different salt than the one specified if it does not contain `$1$`
- by rethrowing the prefix error, it seems to be used in the Javascript engine so the user needs a message that their salt is missing the `$1$` prefix

Reply:
we can probably remove AIX support soon

Reply:
done! https://github.com/Normation/rudder/pull/6004

Reply:
Thanks ! I updated to 1.17.1 :white_check_mark: